sql/mob_spawn_points.sql

```diff
@@ -8057,30 +8057,30 @@ INSERT INTO `mob_spawn_points` VALUES (16929085,'Temenos_Aern','Temenos Aern',12
 INSERT INTO `mob_spawn_points` VALUES (16929086,'Temenos_Aern','Temenos Aern',108,560.000,4.500,-404.000,64);
```

Comment:
i had 16929082 as another rdm on my runs

Reply:
16929082 is a RDM in this PR too.  (It already was one; this PR didn't change it.)

Reply:
i did not see it here and still don't, but i'm sure i'm just missing something - was looking where it jumps from 16929081 to 16929086. not a comment on the pr, but the seemingly missing data. where is that spawn??? ill look around.

Reply:
I believe this is just a side effect of viewing the diff of a large file on github.  It's only showing you the edited lines plus a few on each side, and leaving out many of the unchanged lines between the edits.